### ISSUE TITLE
(WAVE) brak alt textów, missing linki, złą nazwa strony

### ISSUE THREAD

**Issue by vcm1001** [SzymonPajzert/koryta#393]
kilka obserwacji z wtyczką [WAVE od Web AIM](https://wave.webaim.org/) do sprawdzania accessibility (to wpływa [m.in](http://m.in/). na ranking w wyszukiwarkach, więc to nie jest czysta kosmetyka)

- in general brakuje alt textów w obrazkach na stronie; alt text opisuje co się znajduje na obrazku, dzięki temu ludzie z czytnikami ekranu i google w google grafice wie co pokazuje obrazek

<img width="580" height="175" alt="Image" src="https://github.com/user-attachments/assets/db4005fc-e685-47dd-8f2f-2721cac38c9b" />

- empty linki pod logo i pod ikonkami w navie

<img width="262" height="408" alt="Image" src="https://github.com/user-attachments/assets/282dd2ed-8db7-41f6-8371-4bf150068bbe" />

<img width="315" height="151" alt="Image" src="https://github.com/user-attachments/assets/7f3446a0-21a6-4477-988f-e09d97fd3ba8" />

- brak tytułu / niedobry tytuł strony

<img width="388" height="465" alt="Image" src="https://github.com/user-attachments/assets/c03cb573-060c-4fc0-9564-bca6647c6211" />
